docs/starters.yml

```diff
@@ -6301,3 +6301,19 @@
     - Sitemap Generation
     - XML Sitemaps
     - Progressive Web Ap
+- url: https://gatsby-typescript-eslint-prettier-starter.netlify.app/
+  repo: https://github.com/Tielem/gatsby-typescript-eslint-prettier-starter
+  description: This Gatsby starter is an adaptation of the default Gatsby starter with TypeScript, ESlint and Prettier added and pre-configured, bringing you everything you need to get up and running with Gatsby in a type-safe and style-safe way.
+  tags:
```

Comment:
There is a list of available tags for starters here: https://github.com/gatsbyjs/gatsby/blob/master/docs/categories.yml Could you update these to remove the ones not included in there under `starters`? The features bullet points can have whatever text you'd like, but the tags are restricted so we keep a smaller list to filter by in the showcase.